src/components/SearchBar.tsx

```diff
@@ -95,7 +95,9 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
                 this.clearInput();
                 break;
             default:
-                this.searchInputField.focus();
+                if (event.target.className == "SearchInputText") {
+                    this.searchInputField.focus();
```

Comment:
the key handlers are global. codeblock has a text field, any keystroke will get the search in focus. this change should get rid of that